src/inverseRobot.js

```diff
@@ -8,6 +8,27 @@
 
 function inverseRobot(robot) {
   // write code here
+  let tempKyesRobot = {};
```

Comment:
The variable name 'tempKyesRobot' seems to be a typo. Consider renaming it to 'tempKeysRobot' for better readability.